modules/meshrenderinggl/include/modules/meshrenderinggl/processors/rasterizationrenderer.h

```diff
@@ -144,8 +144,7 @@ class IVW_MODULE_MESHRENDERINGGL_API RasterizationRenderer : public Processor {
 
     std::shared_ptr<ImageInport> imageInport_;
     ImageOutport outport_;
-
-    BoolProperty forceOpaque_;
+    Image intermediateImage_;

```

Comment:
This could also be an ImageGL I think. but some of the convenience methods might not exits...

Reply:
Exactly. Is more of a hassle and I have to copy code from the glutils. Hope it's not too much of an overhead?